src/compiler/emitter.ts

```diff
@@ -2138,6 +2138,72 @@ module ts {
             function getTemplateLiteralAsStringLiteral(node: LiteralExpression): string {
                 return '"' + escapeString(node.text) + '"';
             }
+            
+            function emitDownlevelRawTemplateLiteral(node: LiteralExpression) {
+                var text = getSourceTextOfNodeFromSourceFile(currentSourceFile, node);

```

Comment:
Oh.  Is this because it's supposed to be 'raw'?  If so, can you comment here the difference and how this shoudl behave on different inputs?  Thanks!